tez-runtime-library/src/main/java/org/apache/tez/runtime/library/common/shuffle/orderedgrouped/ShuffleScheduler.java

```diff
@@ -1015,51 +1019,63 @@ boolean isShuffleHealthy(InputAttemptIdentifier srcAttempt) {
     final float MIN_REQUIRED_PROGRESS_PERCENT = minReqProgressFraction;
```

Comment:
if you touch this codepath, don't forget to remove these lines of redeclaring stuff:
```
    final float MIN_REQUIRED_PROGRESS_PERCENT = minReqProgressFraction;
    final float MAX_ALLOWED_STALL_TIME_PERCENT = maxStallTimeFraction;
```
I don't get what's the purpose...I guess they are leftovers from a code that needed final variables (in-place Runnables or whatever)